target/linux/pistachio/modules.mk

```diff
@@ -175,3 +175,49 @@ endef
 
 $(eval $(call KernelPackage,nhc_udp))
 
+define KernelPackage/uccp420wlan
+  SUBMENU:=$(WIRELESS_MENU)
+  TITLE:=Imagination wireless driver
+  KCONFIG:=CONFIG_UCCP420WLAN
+  KCONFIG:=\
+    CONFIG_UCCP420WLAN=y \
+    CONFIG_CFG80211=y \
+    CONFIG_CFG80211_DEVELOPER_WARNINGS=n \
+    CONFIG_CFG80211_REG_DEBUG=n \
+    CONFIG_CFG80211_DEFAULT_PS=y \
+    CONFIG_CFG80211_DEBUGFS=y \
+    CONFIG_CFG80211_INTERNAL_REGDB=n \
+    CONFIG_CFG80211_WEXT=y \
+    CONFIG_NL80211_TESTMODE=y \
+    CONFIG_MAC80211=y \
+    CONFIG_MAC80211_RC_MINSTREL=y \
+    CONFIG_MAC80211_RC_MINSTREL_HT=y \
+    CONFIG_MAC80211_RC_MINSTREL_VHT=n \
+    CONFIG_MAC80211_MESH=n \
+    CONFIG_MAC80211_LEDS=y \
+    CONFIG_MAC80211_DEBUGFS=y \
+    CONFIG_MAC80211_DEBUG_MENU=y \
+    CONFIG_MAC80211_NOINLINE=n \
+    CONFIG_MAC80211_VERBOSE_DEBUG=y \
+    CONFIG_MAC80211_MLME_DEBUG=n \
+    CONFIG_MAC80211_STA_DEBUG=n \
+    CONFIG_MAC80211_HT_DEBUG=n \
+    CONFIG_MAC80211_OCB_DEBUG=n \
+    CONFIG_MAC80211_IBSS_DEBUG=n \
+    CONFIG_MAC80211_PS_DEBUG=n \
+    CONFIG_MAC80211_TDLS_DEBUG=n \
+    CONFIG_MAC80211_DEBUG_COUNTERS=n \
+    CONFIG_MAC80211_HWSIM=m \
+    CONFIG_ATH_CARDS=n \
+    CONFIG_MWIFIEX=n \
+    CONFIG_RSI_91X=n \
+  FILES:=$(LINUX_DIR)/drivers/net/wireless/uccp420wlan/uccp420wlan.ko
+  DEPENDS:=+kmod-cfg80211
+  AUTOLOAD:=$(call AutoLoad,30,$(FILES))
```

Comment:
I think this goes directly into the file in /etc/modules.d/30-uccp420wlan - so it should just be the name of the module?